test/extended/util/kcp_general.go

```diff
@@ -0,0 +1,16 @@
+package util
```

Comment:
i think it should be renamed to kcp_common.go may be ?

Reply:
util may be generic IMO, https://github.com/kcp-dev/kcp/blob/main/test/e2e/framework/util.go even uses `framework` : )

Reply:
But i think kcp_general is not a good naming as far i know, we could check with other experts too, it can be kcp_generic.go or common.go, it has better sounding.

Reply:
`_`naming rule has no issuses you could check https://github.com/kcp-dev/kcp/blob/main/pkg/authorization/system_crd_authorizer.go . Just for golang functions name better without `_`. I could change it to `kcp_common.go` or some others if our guys all agree.

Reply:
@Phaow i am not talking about the _ naming rule here, rather talking about the name itself, instead of kcp_general.go it could simply be kcp_common.go or kcp_generic.go

Reply:
ok. I'll change it better to  `kcp_common.go`.

Reply:
Thank you !!